### DISCUSSION TITLE
Releases Draft not showing anymore

### DISCUSSION THREAD

**Discussion by martylamoureux** [community/community#59531] [Repositories]
### Select Topic Area

Bug

### Body

Hello There,

I cannot see my releases drafts anymore (it was working yesterday).

I'm using it with electron-builder which creates the draft and upload assets, but I was not able to see it.
I thought it may be a bug from the electron-builder publish feature, but creating a draft from GitHub GUI is the same. The release draft I just created is not visible in my repo.

I'm using the new UI beta btw, I don't know if it matters.

# Steps to reproduce

- Go to Releases
- Draft a new dummy release
- Save Draft
- Go back to Releases, the drafted release doesn't appear

**Comment by edwarddoh** [comment]
I have same issue on my repo. not just draft, but also regular releases are.

**Comment by piotrooo** [comment]
Same here.
Drafts created manually and through GitHub Action (release-drafter).

**Comment by aonez** [comment]
Same here, draft and releases.
No v1.3.1 showing here: https://github.com/aonez/Keka/releases
It was showing this morning though, and is still available using a direct link: https://github.com/aonez/Keka/releases/tag/v1.3.1
